processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -353,7 +362,8 @@ protected abstract Integer addToFacts(
       InputRow row,
       AtomicInteger numEntries,
       TimeAndDims key,
-      ThreadLocal<InputRow> in
+      ThreadLocal<InputRow> rowContainer,
```

Comment:
Is this actually needed as a parameter?  Can we just create the Supplier<InputRow> here?